class.jetpack-connection-banner.php

```diff
@@ -25,6 +25,10 @@ static function init() {
 	 */
 	private function __construct() {
 		add_action( 'current_screen', array( $this, 'maybe_initialize_hooks' ) );
+
+		if ( \Automattic\Jetpack\Constants::is_true( 'JETPACK_SHOULD_USE_CONNECTION_IFRAME' ) ) {
```

Comment:
Could we avoid the leading backslash in the future? Omitting it can be helpful for testing, while adding it adds little to no value.